bin/util/transifex.js

```diff
@@ -697,6 +697,7 @@ const deletePartialTranslation = ({ translated, parent }) => {
 };
 
 const validateTranslation = (locale) => ({ source, translated, path }) => {
+  if (translated == null) return;
```

Comment:
`translated == null` means that it is a linked locale message that has not been translated. If `translated == null`, then there's nothing to validate, and we immediately return: the function could throw an error otherwise. I'm actually a bit surprised that this hasn't come up before, because there are a few places in this file that deal with untranslated linked locale messages. Before v1.2, the only place we used linked locale messages was the audit log actions. Those tend to be simple strings near the top of the list, so maybe actually they were all translated? 🤔 In any case, I've glanced over some of the logic related to untranslated linked locale messages, and I think things should work (other than this line). There are untranslated linked locale messages in German, but `de.json` looks OK and is correctly missing linked locale messages that are untranslated. `cs.json` has more linked locale messages than `de.json`, because all of them have been translated.